src/main/scala/dxWDL/base/Utils.scala

```diff
@@ -102,6 +102,11 @@ object Utils {
     sb.toString
   }
 
+  // This one seems more idiomatic
+  def readFileContent2(path: Path): String = {
```

Comment:
Sure, just replace the old implementation then. This isn't used in performance contexts anyway.

Reply:
I will eventually, but readFileContent is used in a lot of places, so I'll do it in a separate PR.